src/components/FixedGrid/FixedGridColumn.tsx

```diff
@@ -15,6 +15,12 @@ export interface FixedGridColumnProps extends NestableBaseComponentProps {
    * The number of pixels wide this column should be. Ignored if `fixed` is not set to true.
   */
   width?: number;
+
+  /**
+   * The vertical alignment of the column's content
```

Comment:
This sentence should end with a period.

Reply:
can we add a linter to enforce that?

Reply:
It'd be nice, but I don't know of a rule that'd do this. We'll have to rely on people looking at other descriptions and trying to keep things consistent.